libcfnet/protocol_version.c

```diff
@@ -29,6 +29,10 @@ ProtocolVersion ParseProtocolVersionPolicy(const char *const s)
     {
         return CF_PROTOCOL_TLS;
     }
+    else if (StringEqual(s, "3") || StringEqual(s, "cookie"))
+      {
```

Comment:
```suggestion
    {
```

Reply:
ack, fixed, why doesnt my emacs indent right for our c code?